examples/simple/versions.tf

```diff
@@ -0,0 +1,10 @@
+terraform {
+  required_version = ">= 1.6"
+
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = "~> 5.38"
```

Comment:
```suggestion
      version = ">= 5.38"
```

Reply:
This is not the module's version requirement. This is example code. No need to be a more liberal operator.